pkgs/top-level/all-packages.nix

```diff
@@ -4891,6 +4891,10 @@ with pkgs;
 
   wlprop = callPackage ../tools/wayland/wlprop { };
 
+  hyprevents = callPackage ../tools/wayland/hyprevents { };
+
+  hyprprop = callPackage ../tools/wayland/hyprprop { };
+
```

Comment:
Use the new [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format so you don't have to edit this file